### DISCUSSION TITLE
Can I create links with 'target="_blank"' in Markdown?

### DISCUSSION THREAD

**Discussion by phannhat17** [11ty/eleventy#2280] [Q&A] [answered]
I've try to use html like 

`<a href="http://example.com/" target="_blank">Hello, world!</a>
`
but it does not work

**Comment by cfjedimaster** [comment]
How did it not work? MD should let you enter any HTML you want. I do think if you tab it over it may escape it as code, but if you didn't, it should just work. Maybe share the entire template?

**Comment by pdehaan** [comment, marked as answer]
Worked for me in a simple test case (eleventy@1.0.0, macOS):

```md
---
title: 2280
---

# {{ title }}

<a href="http://example.com/" target="_blank">Hello, world!</a>

- bulleted
- list
```

### OUTPUT

```html
<h1>2280</h1>
<p><a href="http://example.com/" target="_blank">Hello, world!</a></p>
<ul>
	  <li>bulleted</li>
	  <li>list</li>
</ul>
```

**Reply by pdehaan** [reply]
OH! I think it's an indentation "problem".
My test case is basic, but if you indent content by 4 spaces in Markdown, it think's its a `<pre><code>` block, and will do something like this:

```html
<pre><code>&lt;a href=&quot;http://example.com/&quot; target=&quot;_blank&quot;&gt;Hello, world!&lt;/a&gt;
</code></pre>
```

I think we have a ["pitfall"](https://www.11ty.dev/docs/languages/markdown/#there-are-extra-and-in-my-output) for this exact issue (via the [Markdown template](https://www.11ty.dev/docs/languages/markdown/#there-are-extra-and-in-my-output) page).
TLDR (but please R) is that you need to create your own custom [markdown-it library instance](https://www.11ty.dev/docs/languages/markdown/#optional-set-your-own-library-instance) and disable code blocks:

```js
  const markdownIt = require("markdown-it");
  const mdOptions = {
    html: true,
    breaks: true,
    linkify: true
  };
  eleventyConfig.setLibrary("md", markdownIt(mdOptions).disable("code"));
```

**Reply by phannhat17** [reply]
I have had success while using markdown-it. Thank you very much!
